docs/extensions.md

```diff
@@ -22,7 +22,7 @@ Some XDM schemas might be declared non-extensible, which means that any property
 
 XDM generally separates two kinds of properties:
 
-1.  Standard properties are part of the XDM specification, and follow a pattern of `prefix:name`. For instance, `repo:assetID` is the unique identifier of assets in XDM.
+1.  Standard properties are part of the XDM specification, and follow a pattern of `prefix:name`. For instance, `repo:id` is the unique identifier of assets in the content repository.
```

Comment:
replace "the content" with "a" (since we're talking about any type of repo, anywhere)